python/lsst/ts/standardscripts/auxtel/daytime_checkout/slew_and_take_image_checkout.py

```diff
@@ -0,0 +1,225 @@
+
+    Notes
+    -----
+    **Checkpoints**
+
+    - "Bias Frame Verification": Before taking bias frame.
+    - "Slew and take image 1/2": Before slewing to first target. Dome and
+    Sidereal tracking are enabled. Target will track for three minutes.
+    - "Slew and take image 2/2": Before slewing to second target. Dome and
+    Sidereal tracking are enabled. Target will track for three minutes.
+    - "Stopping tracking, parking Telescope and Dome...": Before parking the
+    telescope and dome at end of tests. Final Checkpoint.
+
+    **Details**
+
+    This script is used to perform the daytime checkout of the full AuxTel
+    system to ensure it is ready for nighttime operation. It builds on previous
+    checkouts and performs Telescope and Dome slews and verification images
+    as well as tracking. The Telescope and Dome are left in their parked
+    positions at the end of the chckout and the mirror covers remain closed
+    during the entire test.
+    """
+
+    def __init__(self, index=1):
+
+        super().__init__(
+            index=index,
+            descr="Execute daytime checkout of AT and LATISS.",
+        )
+
+        # Instantiate latiss. We need to do this after the call to
+        # super().__init__() above. We can also pass in the script domain and
+        # logger to both classes so log messages generated internally are
+        # published to the efd.
+        self.latiss = LATISS(domain=self.domain, log=self.log)
+        self.atcs = ATCS(domain=self.domain, log=self.log)
+
+    @classmethod
+    def get_schema(cls):
+        return None
+
+    async def configure(self, config):
+        # This script does not require any configuration
+        pass
+
+    def set_metadata(self, metadata):
+        """Set estimated duration of the script."""
+
+        metadata.duration = 10.0 * 60  # Approximate 5min to completion.
+
+    async def run(self):
+
+        await self.assert_feasibility()
+
+        # Ensure mirrors and vents are closed for safer operations
+        await self.atcs.close_m1_cover()
+        await self.atcs.close_m1_vent()
+
+        # Enable Dome following to allow dome to slew with telescope
+        await self.atcs.enable_dome_following()
+
+        # Bias Verification, start with checkpoint for observer
+        await self.checkpoint("Bias Frame Verification")
+
+        # Turn on ATAOS correction(s)
+        cmd = await self.atcs.rem.ataos.cmd_enableCorrection.set_start(
+            m1=True, hexapod=True, atspectrograph=True
+        )
+        cmd_time = get_topic_time_utc(cmd)
+        self.log.info(f"ATAOS corrections enabled -- {cmd.result} at {cmd_time} UT")
+
+        # Now slew to a new position and start tracking a siderial target.
+        start_az = 45
+        start_el = 45
+        start_rot = 0
+        coord = self.atcs.radec_from_azel(az=start_az, el=start_el)
+        await self.atcs.slew_icrs(
+            coord.ra,
+            coord.dec,
```

Comment:
I think it would be good to provide the target name. By default it is going to be `slew_icrs` I think you could use something like 

```
target_name = f"DaytimeCheckout001"
```

Reply:
Change adopted.